tests/test_username_api.py

```diff
@@ -73,6 +76,11 @@ def test_taken_instagram_username(self, user):
   def test_taken_twitter_username(self, user):
     assert_response(self.app, 'twitter', user, 200)
 
+  @pytest.mark.parametrize('user', data['facebook']['taken_usernames'])
+  def test_taken_github_username(self, user):
```

Comment:
Copy/paste issue – should be `test_taken_facebook_username`

Reply:
My bad.  I should have spotted this.   All the more reason for https://github.com/manu-chroma/username-availability-checker/issues/36